lib/ReactViews/StandardUserInterface/StandardUserInterface.jsx

```diff
@@ -352,9 +370,14 @@ const StandardUserInterface = observer(
                       animationDuration={animationDuration}
                     />
                     <main>
-                      <ExplorerWindow
-                        terria={terria}
-                        viewState={this.props.viewState}
+                      <Route
+                        path={[CATALOG_MEMBER_ROUTE, CATALOG_ROUTE, "*"]}
```

Comment:
Do we need to match "*"?

Reply:
Yes, we also want to mount the ExplorerWindow when on "*" because of the animations etc